tests/integration/goldens/asset/samples/generated_samples/snippet_metadata_v1_asset_v1.json

```diff
@@ -0,0 +1,1048 @@
+{
+  "snippets": [
+    {
+      "clientMethod": {
+        "async": true,
+        "method": {
+          "fullName": "AnalyzeIamPolicyLongrunning",
+          "service": {
+            "shortName": "AssetService"
```

Comment:
And fullName would be the service name qualified by the proto package.
Fine if this is one the ones you are adding later.